README.md

```diff
@@ -87,13 +58,44 @@ DEV_REGISTRY=myhost:5000 make push
 ```
 
 ### Kubernetes interactions
-* `DEV_REGISTRY=quay.io IMAGE_PREFIX=myaccount make kube-create`: Does a sed replacement of the image name and creates a tmp manifest file that we will use to deploy logger component to kubernetes. This will also start 2 `fluentd` daemonsets.
-* `make kube-delete`: This will remove all the logger components from the kubernetes cluster.
-* `DEV_REGISTRY=quay.io IMAGE_PREFIX=myaccount make kube-replace`: This will rebuild the binary and image, push it to the remote registry, and then replace the running components with the new version.
+* `make install` - Install the recently built docker image into the kubernetes cluster
+* `make upgrade` - Upgrade a currently installed image
+* `make uninstall` - Uninstall logger from a kubernetes cluster
+
+### Architecture Diagram
+```
```

Comment:
This diagram needs some whitespace corrections so boxes align cleanly.
![screen shot 2016-08-26 at 11 33 50 am](https://cloud.githubusercontent.com/assets/73019/18014639/18a8ad1c-6b81-11e6-83cb-5b4e3997b467.png)